src/features/auth/constants/auth.keys.ts

```diff
@@ -0,0 +1 @@
+export const AUTH_ACCESS_TOKEN = "";
```

Comment:
The key cannot be empty, so let's add a default key and a todo so we know that it should be updated when used on a project
```suggestion
// TODO: Update to use name of project
export const AUTH_ACCESS_TOKEN = "admin-panel-tokens";
```